escrow/project/contracts/escrow-contract/tests/utils/setup.rs

```diff
@@ -15,13 +15,13 @@ abigen!(
     ),
     Contract(
         name = "MyAsset",
-        abi = "./contracts/escrow-contract/tests/artifacts/asset/out/debug/asset-abi.json"
+        abi = "./contracts/test-artifacts/asset/out/debug/asset-abi.json"
```

Comment:
As mentioned in another PR, this should be moved back into tests or updated to use the SDK coin/asset generation API

Reply:
This is required due to changes with nested projects and forc workspaces. An error is produced if the test artifacts are kept in their old places.

Reply:
Have re-written to use SDK